spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/kafka/ConcurrentKafkaListenerContainerFactoryConfigurer.java

```diff
@@ -101,6 +101,11 @@ private void configureContainer(ContainerProperties container) {
 				.to(container::setAckTime);
 		map.from(properties::getPollTimeout).whenNonNull().as(Duration::toMillis)
 				.to(container::setPollTimeout);
+		map.from(properties::getClientId).whenNonNull().to(container::setClientId);
```

Comment:
Hi, I could not find the method setClientId for my ContainerProperties, I think it caused some issues

Reply:
If you do, you are overriding the managed version for Spring Kafka in your application. Please remove the override or upgrade to the snapshot for the time being. Spring Kafka `2.1.1.RELEASE` will have this property.